### ISSUE TITLE
Only few streaming lines on windows in UVC_FRAME_FORMAT_YUYV

### ISSUE THREAD

**Issue by ftaiolivista** [pupil-labs/libuvc#22]
Hi,

I'm using your libusb and libuvc with opencv.

My device support UVC_FRAME_FORMAT_YUYV mode.

On Linux I can get a full frame streaming but on windows i can get only few lines of video streaming, but that few lines are working at least :)

![screenshot_20171110_115932](https://user-images.githubusercontent.com/150721/32655797-7705ba1e-c60f-11e7-97b0-d8f7f1ab0e69.jpg)

Any idea on how I can fix that?

I'm on windows 10, device driver libusbK (v3.0.7.0). Tested with many device, same results.

Thank you for your great work on windows platform.

**Comment by mkassner** [comment]
I m not sure. We made the port work for mjepg only. We dont use yuv transport. Maybe ask in themain repo of this fork?

**Comment by ftaiolivista** [comment]
It's this code on stream.c, if I comment it it works :)). But why you have added it? I think there must be a reason :)

```
    // our way: estimate it:
    size_t bandwidth = frame_desc->wWidth * frame_desc->wHeight / 8 * bandwidth_factor; //the last one is bpp default 4 but we use if for compression, 2 is save, 1.5 is needed to run 3 high speed cameras. on one bus.
    bandwidth *= 10000000 / strmh->cur_ctrl.dwFrameInterval + 1;
    bandwidth /= 1000; //unit
    bandwidth /= 8; // 8 high speed usb microframes per ms
    bandwidth += 12; //header size
    config_bytes_per_packet = bandwidth;
```

**Comment by mkassner** [comment]
Yeah. This is for mjpeg bandwith calculation. You should leave it out as you did for yuv.
